### PR TITLE
add tripadvisor2mimir

### PR DESCRIPTION
**Pull request by remi-dupre**
Add a new binary to index TripAdvisor POIs.

### CODE REVIEW

File: src/sources/tripadvisor/mod.rs [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+                                quick_xml::de::from_str(&raw).expect("failed to parse properties");
+
+                            convert::build_poi(property, geofinder.as_ref())
+                                .map_err(|err| *local_errors.entry(err).or_insert(0) += 1)
+                                .ok()
+                        })
+                        .collect();
+
+                    if !local_errors.is_empty() {
+                        warn!("parsion errors for current block: {:?}", local_errors);
```
Comment (fatal69100) [inline src/sources/tripadvisor/mod.rs:48]:
```suggestion
                        warn!("parsing errors for current block: {:?}", local_errors);
```

Comment (fatal69100) [inline src/sources/tripadvisor/mod.rs:48]:
```suggestion
                        warn!("parsing errors for current block: {:?}", local_errors);
```

Comment (remi-dupre) [reply src/sources/tripadvisor/mod.rs]: I've moved this logic in the binary to collect errors in a unified data-structure and display the stats only once.

**Comment by remi-dupre** [comment]
@fatal69100, do you think we should merge this PR as is?

This would allow to open other independant PRs for:
 - updating mimir dependency to latest version (the config format changed a bit, and mimir2 has been renamed to mimir)
 - iteration over tripadvisor2mimir, which lacks some features and tests, of course we would have to keep in mind that it is not production ready

**Comment by fatal69100** [comment]
I agree. I add an issue to don't forget it

**Comment by remi-dupre** [comment]
I let you put you seal of approval if you want to have a last look before :wink:

**Review by fatal69100 — APPROVED** [review]

**Comment by fatal69100** [comment]
Can we merge it in order to integrate automatically PropertyList index to ES dev with argo?

**Pull request merged**